modules/controller/index.js

```diff
@@ -70,21 +70,16 @@ Controller.prototype.read = function (opts) {
     opts = {};
   }
   var source = this.source;
-  var type = source.typeName();
   var filters = this._filters.bind(this);
   var relations = this._relations.bind(this);
   // allow a custom responder to receive the payload
   var respond = opts.responder || responder;
   return function (request, response, next) {
-    var validRelations = relations(request).concat(opts.include || []);
     source.read({
       filters: filters(request),
-      relations: validRelations
+      relations: relations(request).concat(opts.include || [])
     }, function (err, data) {
-      var payload = readResponse(err, data, _.extend({}, opts, {
```

Comment:
We still need the opts object here, the type needs to be passed in so the responder knows what namespace to put the data under. I'm extending because I thought there might be a future use-case for wanting the other options too, but I'd be game for just passing in:
```js
{ 
  type:type,
  relations: validRelations
}
```